keras_cv/layers/preprocessing/grid_mask.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers, backend
+
+
+class GridMask(layers.Layer):
+    """GridMask class for grid-mask augmentation. The expected images should be [0-255] pixel ranges.
+    
+    Args:
+        ratio: Grid mask ratio i.e if 0.5, grid and spacing will be equal.
+            Defaults to 0.5. It ranges from 0.0 to 1.0.
+        gridmask_size_ratio: Grid mask size, grid to image size ratio.
+            Defaults to 0.5. It ranges from 0.0 to 1.0.
+        rate: Float between 0 and 1. The probability of augmenting an input.
```

Comment:
We don't need to support a rate parameter for this layer.  Thanks

Reply:
https://github.com/keras-team/keras-cv/issues/100#issuecomment-1029965632